action-allowedlist/src/dependencies.psm1

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This was previously a plain script, but used like a module. So I renamed it as a module.